classes/person.rb

```diff
@@ -0,0 +1,37 @@
+require_relative './nameable'
+
+class Person < Nameable
+  @all_persons = []
+
+  attr_reader :id, :rentals
+  attr_accessor :name, :age
+
+  def initialize(age, name, parent_permission)
+    super()
+    @id = Random.rand(1..1000)
+    @name = name
+    @age = age
+    @parent_permission = parent_permission
+    @rentals = []
+    self.class.all_instances.push(self)
+  end
+
```

Comment:
- [ ] There is a bug in your app. whenever I try to create a student or a teacher. it gives me an error that goes back to line 16 of this class.
I have checked that your books are getting created, but neither the teacher or student getting created, so I cant check the rentals as well. kindly consider fixing this bug. I am attaching a video for demonstration.

https://github.com/sameedmohsin28/OOP-school-library/assets/96683843/6331f1c5-2568-4ea0-b964-aef230384400